davinci-dome/src/Download.ts

```diff
@@ -21,13 +21,20 @@ export interface DaVinciOutput {
 function extractNodeFile(output: DaVinciOutput): string {
     const rows: string[][] = []
     rows.push(["index", "x", "y", "z"])
-    output.joints.forEach((joint, index) => rows.push([
-        (index + 1).toFixed(0),
-        csvNumber(joint.point.x), csvNumber(joint.point.z), csvNumber(joint.point.y),
-    ]))
+    output.joints
+        .filter(joint => joint.position !== JointPosition.Below)
```

Comment:
ignore the joints which are below

Reply:
Ik sta te trappelen om RFEM zo in te laden :)